templates/4_Post_Clustering.ipynb

```diff
@@ -13,7 +13,7 @@
   },
```

Comment:
<p>This looks like the old version of the notebook, we want to keep the one that's currently in main</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/794/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>